### DISCUSSION TITLE
Need help with configuring the mod

### DISCUSSION THREAD

**Discussion by Pirogiran** [null2264/CobbleGen#10] [Q&A] [answered]
I'm trying to make Andesite, Granite, Diorite, Calcite, Tuff, End Stone and Netherrack generators.
But i'm doing something wrong...
Only Netherrack working.
I'm feeling surprisingly stupid.
[cobblegen.txt](https://github.com/null2264/CobbleGen/files/10047490/cobblegen.txt)

**Comment by null2264** [comment, marked as answer]
To add new custom generator variant (with cobbleGen method), you add it inside of "cobbleGen"
So instead of doing this:
```json
	"customGen": {
		"cobbleGen": {
			"minecraft:end_stone": [
				{
					"id": "minecraft:end_stone",
					"weight": 100.0,
				}
			]
		}
	},
	"customGen": {
		"cobbleGen": {
			"minecraft:granite": [
				{
					"id": "minecraft:granite",
					"weight": 100.0,
				}
			]
		}
	},
```

You do this:
```json
	"customGen": {
		"cobbleGen": {
			"minecraft:end_stone": [
				{
					"id": "minecraft:end_stone",
					"weight": 100.0,
				}
			],
			"minecraft:granite": [
				{
					"id": "minecraft:granite",
					"weight": 100.0,
				}
			]
		}
	},
```

**Reply by Pirogiran** [reply]
Oh, i see...
I spent about two hours to figure out what was wrong.
Thank you!
